app/models/api/openid_connect/id_token.rb

```diff
@@ -0,0 +1,52 @@
+module Api
+  module OpenidConnect
+    class IdToken < ActiveRecord::Base
+      belongs_to :authorization
+
+      before_validation :setup, on: :create
+
+      default_scope { where("expires_at >= ?", Time.zone.now.utc) }
```

Comment:
**Edit**: Nope, actually, @theworldbright is right. This field is represented as the number of seconds from 01/01/1970.